tpu-client/src/nonblocking/tpu_client.rs

```diff
@@ -503,19 +503,21 @@ where
         Self::new_with_connection_cache(rpc_client, websocket_url, config, connection_cache).await
     }
 
-    /// Try to create LeaderTpuService
+    /// Create LeaderTpuService
     /// Retries until successful or timeout is reached
-    async fn try_create_leader_tpu_service(
+    async fn create_leader_tpu_service(
```

Comment:
If `rpc_client` is not `self`, why is this a method rather than just a top level function in this module?
It would decrease indentation and would remove `Self::` from the call site.
This function seems like a private implementation detail.
The main difference is that a method is in the type namespace, but if the method is not public, it does not seem to matter.
Or do you expect it to be used from elsewhere?